Makefile

```diff
@@ -116,6 +116,11 @@ check.dev-container-healthy:
 		exit 1; \
 	fi
 
+run-dsadmin: ## Runs DSAdmin inside the dev-server container.
+	@docker compose up dev-server -d; \
```

Comment:
Remove this line. Let's have devs spin up the dev-server and run `run-dsadmin` to access the ds-admin.

Reply:
done!